pkg/plugins/golang/v3/scaffolds/internal/templates/makefile.go

```diff
@@ -108,8 +110,9 @@ fmt: ## Run go fmt against code.
 vet: ## Run go vet against code.
 	go vet ./...
 
+ASSETS_DIR = $(shell pwd)/bin/
 test: manifests generate fmt vet envtest ## Run tests.
-	go test ./... -coverprofile cover.out
+	KUBEBUILDER_ASSETS=$(shell $(ENVTEST) use $(K8S_VERSION) --bin-dir $(ASSETS_DIR) -p path) go test ./... -coverprofile cover.out
```

Comment:
Why is a separate assets dir needed? Can't this just use the default, globally configured one? Otherwise projects don't benefit from shared assets.
```suggestion
	KUBEBUILDER_ASSETS=$(shell $(ENVTEST) use $(K8S_VERSION) -p path) go test ./... -coverprofile cover.out
```

Reply:
This is Mac specific - the default dir corresponds to a path like `xxxx/Application Support/io.kubebuilder.envtest/xxx`. A trivial issue with bash - because of the space between directory name it errors (`Support/io.kubebuilder.envtest/k8s/1.21.2-darwin-amd64: No such file or directory`). So I thought it would be better to have a separate assets directory.

Reply:
Sorry, I take it back. double quoting the path works!

Reply:
Just quote the path. You'll have the same issue if a parent of the current working directory has a space-separated name.
```suggestion
	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(K8S_VERSION) -p path)" go test ./... -coverprofile cover.out
```